SECURITY.md

```diff
@@ -7,4 +7,4 @@ See [docs/SECURITY-PROCESS.md](docs/SECURITY-PROCESS.md) for full details.
 If you have found or just suspect a security problem somewhere in curl or libcurl,
 report it on [https://hackerone.com/curl](https://hackerone.com/curl).
 
-We treat security issuse with confidentiality until disclosed controlled and responsibly.
+We treat security issues with confidentiality until disclosed and controlled.
```

Comment:
I think "disclosed responsibly" would be a better end of that sentence. It's not that the issue is "controlled" separately when it gets disclosed.

Reply:
ok